lib/node_modules/@stdlib/math/base/special/gcdf/benchmark/julia/REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 1.5
```

Comment:
I don't think we need this file for the single-precision implementation. This can be removed.